src/odemis/acq/stream/_sync.py

```diff
@@ -576,8 +576,12 @@ def _get_center_pxs(self, rep, sub_shape, datatl):
         # Note: we don't rely on the MD_PIXEL_SIZE, because if the e-beam was in
         # spot mode (res 1x1), the scale is not always correct, which gives an
         # incorrect metadata.
-        pxsx = self._getPixelSize()
-        pxs = pxsx / sub_shape[0], pxsx / sub_shape[1]
+        pxs = self._getPixelSize()
+        if not isinstance(pxs, tuple):
+            pxs = pxs, pxs
+
+        pxs = pxs[0] / sub_shape[0], pxs[1] / sub_shape[1]
```

Comment:
Let's just update `_getPixelSize()` to always return a tuple, and adjust "all" the callers. You've found 2 of them, but there is a third one in the file.